main.tf

```diff
@@ -40,6 +42,13 @@ resource "azurerm_storage_account" "storage_account" {
       delete_retention_policy {
         days = 365
       }
+	  cors_rule {
```

Comment:
Can this block be dynamic depending if the 'allowed_origins' has any inputs please